client/src/components/user/UserRouter.js

```diff
@@ -44,7 +44,7 @@ const UserRouter = () => {
           });
     };
     checkLoggedIn();
-  });
+  }, [login]);
```

Comment:
Adding an empty array resulted in ESLint warning saying that `login` is not included.
However, I know that login is a constant declared in this component. Therefore, it's not gonna change, right?